### ISSUE TITLE
Easier way to create blank samples

### ISSUE THREAD

**Issue by ml-evs** [the-grey-group/datalab#643]
From discussions with @MJCliffe, it would be nice to add a way to generate a series of samples with random IDs in the UI. In @MJCliffe's case, he actually wants to scrap the "concept" of `item_id` and just use refcodes everywhere, which we could implement as just a simple duplication with `item_id` being the unprefixed refcode.

This could also tie in nicely with the barcode printer ideas.

I'm imagining some third button alongside Add Item, Add Batch for "Create Item" or similar? This could directly add n rows in the table, and if n=1, open the sample page. This might also be where we introduce the concept of planned vs "real" samples that would be useful in DoE applications, or at least some kind of sample "status" (planned, made, failed, ... etc?) Not as sure about this one, but I think we need a way of showing/filtering the distinction in the UI. Thoughts @jdbocarsly?

**Comment by jdbocarsly** [comment]
I like the idea of having a status dropdown. In our group, we  already need a way to indicate failed experiments, so this could knock out two birds with one stone. Statuses could be: "planned", "in progress", "complete", and "failed". Some issues that I imagine will come up are:

1. People may not actually fill it out and it could get confusing.
2. The definition of a "failed" experiment is not well defined. For our current case, I'm thinking of using it for experiments where the reaction just didn't work at all and we didn't get any product. In other cases, we get a product but it just isn't pure or good enough to do much with. I think I wouldn't mark that as "failed", but other people might. I guess it ultimately will be up to the experimenter to be consistent? The free text description is also the best source of truth for anything nuanced like this.

For the Cliffe group, it sounds like they would want to ditch the `item_id` field in the "add item" and "add batch" modals, and then we could just have `item_id` assigned to automatically on creation. Rather than maintaining these changes in a fork, I think this would be a generally useful option:
<img width="491" alt="Screen Shot 2024-03-12 at 8 39 44 AM" src="https://github.com/the-grey-group/datalab/assets/32345545/97cf01a6-e8e9-4fea-93bd-7c8af9583f8b">

This could be done within the batch dialog too.

**Comment by ml-evs** [comment]
Yep, that's exactly what I was thinking of adding. Will see if I can get an MVP working for the Thursday meeting
